math/approximate_pi.cpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file
+ * @brief Implementation to  calculate an estimate of the number π (Pi).
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂